src/SonarScanner.MSBuild/BootstrapperClass.cs

```diff
@@ -51,7 +44,9 @@ public BootstrapperClass(IProcessorFactory processorFactory, IBootstrapperSettin
         {
         }
 
-        public BootstrapperClass(IProcessorFactory processorFactory, IBootstrapperSettings bootstrapSettings, ILogger logger, Func<string, Version> getAssemblyVersionFunc)
+        public BootstrapperClass(IProcessorFactory processorFactory,
+                                 IBootstrapperSettings bootstrapSettings,
+                                 ILogger logger, Func<string, Version> getAssemblyVersionFunc)
```

Comment:
if you move each on its own line, then also put the `Func` on its own line... 

the current layout seems inconsistent